mysite/__init__.py

```diff
@@ -1 +1 @@
-"""Initialize the package when created Django project."""
+"""package have files with settings for the project."""
```

Comment:
It's not a prose. And it's completely wrong. And it's obvious that it's a package (everyone sees that it's a folder with python files it).

Reply:
And you are required to write sentences, they start with uppercase letters.

Reply:
Better: `Blog app.`